_includes/docs/user-guide/install/config.md

```diff
@@ -662,6 +662,18 @@ We will list only main configuration parameters below to avoid duplication of th
          <td>604800</td>
          <td>The parameter to specify TTL(Time To Live) value for Debug Events(DEBUG_CONVERTER, DEBUG_INTEGRATION, DEBUG_RULE_NODE, DEBUG_RULE_CHAIN) records. Value set in seconds. 0 - records are never expired. Default value corresponds to one week.</td>
       </tr>
+      <tr>
+         <td>sql.ttl.rpc.enabled</td>
+         <td>SQL_TTL_RPC_ENABLED</td>
+         <td>true</td>
+         <td>Parameter to enable or disable TTL(Time To Live) for Persistent RPC.</td>
+      </tr>
+      <tr>
```

Comment:
You have added not about two parameters, but the main document does not cover how these parameters are used. Also, the new tenant profile parameter is not present.